### DISCUSSION TITLE
IF YOUR INTERNET BUG OR NOT AVAILABLED ETC

### DISCUSSION THREAD

**Discussion by kael-ux** [Jyben/cs2-mm-server-picker#121] [General]
**Reset firewall rules** 
 
 
![f1](https://github.com/Jyben/csgo-mm-server-picker/assets/85127174/7defd22a-37c2-493d-b209-c4f2f7b40734)

![f2](https://github.com/Jyben/csgo-mm-server-picker/assets/85127174/508c6a28-6224-4ab9-8646-68c6960ced26)

**Comment by allubie** [comment]
does it work?
